epevermodbus/driver.py

```diff
@@ -13,6 +14,21 @@ class EpeverChargeController(minimalmodbus.Instrument):
 
     """
 
+    battery_voltage_control_register_names = [
+        "over_voltage_disconnect_voltage",
+        "charging_limit_voltage",
+        "over_voltage_reconnect_voltages",
```

Comment:
is this a typo here? Should it be `over_voltage_reconnect_voltage`?

Reply:
It should! Nice catch.